src/main/java/org/isf/vaccine/rest/VaccineController.java

```diff
@@ -111,7 +111,7 @@ public VaccineController(VaccineBrowserManager vaccineManager, VaccineMapper vac
      */
     @PostMapping(value = "/vaccines", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity newVaccine(@RequestBody VaccineDTO newVaccine) throws OHServiceException {
-	    log.info("Create vaccine: {}", newVaccine.toString());
```

Comment:
here the `toString()` is actually overridden in `VaccineDTO` class, and I think with the only intention to have this log, but it doesn't seem to be a costant behaviour in OP-6 (there are many contributors) so what to do?

I would get rid of them (also existing the overridden methods) and maybe define a common pattern for all DTO classes (if really needed at a certain point).

Otherwise this PR should not remove what is actually overridden (and so not unnecessary)

Reply:
You don't need to specify `toString()` explicitly to get the overridden behavior.  Given this code:
````
package org.isf;

public class MyString {

	public static void main(String[] args) {
		MyClass myClass = new MyClass();
		System.out.println("myClass: " + myClass);
		System.out.println("myClass.toString(): " + myClass.toString());
	}

	static class MyClass {

		int x = 1;

		@Override
		public String toString() {
			return "This is from toString()";
		}
	}
}
````

The output is:
````
myClass: This is from toString()
myClass.toString(): This is from toString()
````

Reply:
You are right, I got the point now.